### PR TITLE
First Implementation with react-router-dom-v5-compat

### PR DESCRIPTION
**Pull request by auan369**
Mostly okay, expect for one segment in root.tsx:

```
 <Route path="/">
            <ConfigsProvider configJson={{}}>
              <App />
            </ConfigsProvider>
</Route>
```

No able to convert this to CompatRoute, likely due to the CompactRouter component being inside the App component. Tried moving CompactRouter to the root.tsx but causes issues with the navigation.

### CODE REVIEW

File: packages/web/src/root.tsx [hunk 1]
```diff
@@ -20,13 +21,16 @@ const Root: React.FC<Props> = ({ store, history }) => (
...
             <ConfigsProvider configJson={{}}>
               <App />
             </ConfigsProvider>
           </Route>
-          <Route>NOT FOUND</Route>
+          {/* <CompatRoute path="/" configJson={{}}>
+            <App />
+          </CompatRoute> */}
+          <CompatRoute>NOT FOUND</CompatRoute>
         </Switch>
```
Comment (petrvecera) [inline packages/web/src/root.tsx:34]: @auan369  I think we can remove the router from this file `root.tsx` we are not using the route /login , so we can remove all the stuff. The only thin is that we should probably handle 404 route somehow differently? But we can resolve that when we move to react router 6

File: packages/web/src/App.tsx [hunk 2]
```diff
@@ -1,8 +1,7 @@
 import React from "react";
 import { Layout } from "antd";
 import "./App.css";
-import { Route, Switch } from "react-router-dom";
-
+import { Switch } from "react-router-dom";
```
Comment (petrvecera) [inline packages/web/src/App.tsx:4]: I assume we want to import this switch also from "react-router-dom-v5-compat" and completely remove the package "react-router-dom"

File: packages/web/src/root.tsx (hunk 1 above)
Comment (auan369) [reply packages/web/src/root.tsx:34]: Okay will work on this

File: packages/web/src/App.tsx (hunk 2 above)
Comment (auan369) [reply packages/web/src/App.tsx:4]: Yup will work on this as well. Thank you!

**Comment by petrvecera** [comment]
Looks good, you just need to run `yarn fix` so it runs prettier 👍

**Comment by petrvecera** [comment]
I have found little issue with player card but I can fix that :) Thanks!

**Pull request merged**